services/dewey/src/dewey/core.clj

```diff
@@ -122,7 +106,10 @@
     (try+
      (let [{:keys [options arguments errors summary]} (ccli/handle-args svc-info args cli-options)]
        (when-not (fs/exists? (:config options))
-         (ccli/exit 1 (str "The config file does not exist.")))
-       (run (partial config/load-config-from-file (:config options))))
+         (ccli/exit 1 "The config file does not exist."))
+       (when-not (fs/readable? (:config options))
+         (ccli/exit 1 "The config file is not readable."))
```

Comment:
I added this check on readability since it seemed like that would also be useful, not that it'd probably work if it went further anyway.